irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel where everyone is welcome to discuss the game or request support.
```

Comment:
Is this description any better?

Reply:
Might change it to "Main channel for general game and engine discussion or support" in the next commit tho.

Reply:
Remember MT is not a 'game', better to refer to 'Minetest'.

Reply:
I agree that 'everyone is welcome' is not needed.